dh_parameters:
  joint_1: [0.0, 0.05, 0.5, 0.0]
  joint_2: [0.0, 0.0,  0.58, 1.5708]
  joint_3: [1.57, 0.2,  0.4,  0.0]
  joint_4: [1.57, 0.0,  0.1,  0.0]
  joint_5: [1.57, 0.0,  0.3,  0.0]
